gradle/verification-metadata.xml

```diff
@@ -3473,6 +3473,11 @@
             <sha256 value="50e4c7a0d0c68413d3d8587560d56945ac09e7c89c41bd971cd22d76be6f1085" origin="Generated by Gradle"/>
          </artifact>
       </component>
+      <component group="org.bouncycastle" name="bc-fips" version="1.0.2.6">
```

Comment:
Can we remove the 1.0.2.5 version from that file then?

Reply:
Yeah, I always forget to do that.